src/backend/distributed/utils/ruleutils_95.c

```diff
@@ -3398,6 +3398,42 @@ get_utility_query_def(Query *query, deparse_context *context)
 			simple_quote_literal(buf, stmt->payload);
 		}
 	}
+	else if (query->utilityStmt && IsA(query->utilityStmt, TruncateStmt))
```

Comment:
I was a _little_ hesitant to suggest this approach, as this pipeline is largely used for rules; however, the particular version of this function (i.e. here in `ruleutils_95`) will only ever be called by Citus, so I think I feel safe.